highfive/configs/_global.json

```diff
@@ -2,6 +2,12 @@
     "groups": {
         "core": ["@brson", "@nikomatsakis", "@alexcrichton", "@aturon"],
         "crates": ["@alexcrichton"],
-        "doc": ["@steveklabnik"]
+        "doc": ["@steveklabnik", "@GuillaumeGomez"]
+    },
+    "mentions": {
+        "rustdoc-html": {
```

Comment:
rather than hardwiring "rustdoc-html" I'd prefer to add that directory to the config for the rust repo